### DISCUSSION TITLE
how to make the gripper grasp the object toughly

### DISCUSSION THREAD

**Discussion by Yangziqi1920** [bulletphysics/bullet3#3732] [General]
Hello everyone
Now I am doing pybullet with dobot robot, I made my own urdf file through solidworks, but when I want to make my gripper grasp the object tough, but it doesn't work rightly, my griper cannot grasp the object.
I saw many examples such as panda kuka robots, their gripper can rightly grasp the object. I want to know why?
Thanks for your answers!
![1](https://user-images.githubusercontent.com/57791255/91296354-c557c700-e79c-11ea-8f08-b15f02121aff.PNG)
pybullet.setJointMotorControl2(RobotUID,
FINGER_JOINT_NUM,
pybullet.POSITION_CONTROL,
targetPosition=0,
force=self.fingerTipForce)
this is my method.

**Comment by Steven89Liu** [comment]
Could you provide a simple test that i can reproduce your issue? thanks.

**Comment by 201905335** [comment]
Have you solved it?

**Comment by xiaolijz** [comment]
> Hello everyone Now I am doing pybullet with dobot robot, I made my own urdf file through solidworks, but when I want to make my gripper grasp the object tough, but it doesn't work rightly, my griper cannot grasp the object. I saw many examples such as panda kuka robots, their gripper can rightly grasp the object. I want to know why? Thanks for your answers! ![1](https://user-images.githubusercontent.com/57791255/91296354-c557c700-e79c-11ea-8f08-b15f02121aff.PNG) pybullet.setJointMotorControl2(RobotUID, FINGER_JOINT_NUM, pybullet.POSITION_CONTROL, targetPosition=0, force=self.fingerTipForce) this is my method.

Hello, have you solved this problem, I current met the same problem in pybullet, looking for your reply
